ansible/roles/visual-studio/tasks/main.yml

```diff
@@ -4,26 +4,51 @@
 # Install Visual Studio
 #
 
-- when: vs == '2013'
-  block:
-  # TODO: Ensure no other versions are installed
-  - name: install Visual Studio 2013
-    include_tasks: "partials/vs2013.yml"
+- name: install Visual Studio 2013
```

Comment:
Do we still want to test and support 2013?  https://docs.microsoft.com/en-us/visualstudio/releases/2019/servicing#support-for-older-versions-of-visual-studio

Reply:
VS2013 is used to test add-ons for Node.js v8. We should keep the machines available for some time after a Node version goes EOL, but anyway I'd like this to be in the commit history for documentation (we might need to test an old issue or something).